spec/puppet-lint/plugins/check_trailing_comma/check_trailing_comma_spec.rb

```diff
@@ -55,6 +55,17 @@ class { '::apache':
             '/etc/baz.conf', '/etc/baz.conf.d'
              ],
         }
+
+        function myfunc (
+          Mymod::Mytype $arg1,
+          String[1] $arg2,
+        ) >> Mymod::Mytype {
+          notice('foo')
+        }
+
+	if $var =~ Sensitive {
```

Comment:
Can you use spaces here for indenting like the rest of the file?

Reply:
Yes, sorry about that. I wasn't using my normal development environment when I did this patch. Fixed.